<!DOCTYPE html>
<html lang="en">
<head>
  <link href="https://fonts.googleapis.com/css2?family=Nunito&family=Patrick+Hand+SC&display=swap" rel="stylesheet">
  <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.1/css/all.min.css" rel="stylesheet" >
  <link href="site-styles.css" rel="stylesheet">
  <meta charset="UTF-8" />
  <title>sin's cv</title>
  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
  <script type="text/javascript" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script>
  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
</head>

<header class="site-header">
  
  <div class="header-inner site-nav">
      <a href="#" onclick="history.back(); return false;" class="site-nav-title"><i class="fa-solid fa-arrow-left icon-fix-half"></i> back</a>
    <nav class="site-nav">
      <a href="index.html" class="site-nav-title">home</a>
      <a href="about-me.html">about</a>
      <a href="cv.html">cv</a>
      <a href="publications.html">publications</a>
      <a href="other.html">other</a>
    </nav>
  </div>
</header>

<body>
<div class="container">
  <h2 style="text-align: center; margin-top: 0; margin-bottom: 0;" >Curriculum Vitae</h2>
  <h1 style="font-size: 0.8em; text-align: center; margin-top: 0;">
    <a href="#work-experience" style="color: #ffbf00;">work experience</a> · 
    <a href="#education" style="color: #ffbf00;">education</a> · 
    <a href="#academic-works" style="color: #ffbf00;">academic works</a> · 
    <a href="#skills" style="color: #ffbf00;">skills</a> · 
    <a href="#awards" style="color: #ffbf00;">awards</a> · 
    <a href="#teaching" style="color: #ffbf00;">teaching</a> · 
    <a href="#volunteering" style="color: #ffbf00;">volunteer work</a>
  </h1>

<!-- FIX THE LINKS AND TITLES AND STUFF --------------@@@@@@@@@@@@@@@@@@@ -->

<!-- <div class="hz-divider"></div> -->



<!-- work experience -->
<section class="mainfont" id="work-experience">
  <h1>WORK EXPERIENCE</h1>
<div class="hz-divider" style="margin-top: 0rem; margin-bottom: 1.5rem;"></div>
  <section class="cv-entry light-section-half" style="margin-bottom: 1.5rem;">
  <div class="cv-entry-header">
    <span class="cv-entry-title">Doctoral Researcher</span>
    <span class="cv-entry-years">2018-2024</span>
  </div>
  <div class="cv-entry-subtitle-1">
    Molecular cell biology of insect olfactory system
  </div>
  <div class="cv-entry-subtitle-2">
    <i>Reception & Transduction</i> & <i>Drosophila Neurogenetics</i> groups<br>
    Dept. Evolutionary Neuroethology<br>
    Max Planck Institute for Chemical Ecology
  </div>
   <ul class="cv-entry-bullets">
    <li>Lead scientist on multiple projects in insect antennal biology, resulting in <a href="publications.html#publications">5 peer-reviewed publications</a></li>
    <li>Achieved the first single-cell transcriptome of insect antennae (microfluidics-based scRNA-seq)</li>
    <li>Developed probes for non-neuronal cell activity in olfactory tissue, combining transcriptomics, live imaging, neurogenetics, and electrophysiology in <em>Drosophila</em></li>
    <li>Researched diverse topics (V-ATPases, gap junctions, NO-cGMP signaling, non-neuronal cell biology) and fulfilled advisory role on projects in insect ethology, neuroanatomy, <i>Drosophila</i> genetics, etc.</li>
    <li>Active community involvement: <a href="#teaching">teaching</a>, elected PhD representative (3×), outreach (online lectures, podcasts), conference organization, etc.</li>
  </ul>
  </section>

  <section class="cv-entry" style="margin-bottom: 1.5rem;">
  <div class="cv-entry-header">
    <span class="cv-entry-title">Master researcher</span>
    <span class="cv-entry-years">2015-2016</span>
  </div>
  <div class="cv-entry-subtitle-1">
    Cytoskeleton (microtubule) dynamics in sprouting angiogenesis
  </div>
  <div class="cv-entry-subtitle-2">
    Cellular dynamics / Akhmanova lab<br>
    Dept. Cell Biology<br>
    Utrecht University
  </div>
   <ul class="cv-entry-bullets">
    <li>Studied microtubule involvement in blood vessel formation using embedded endothelial spheroids</li>
    <li>Demonstrated impaired angiogenesis in 3D endothelial organoid model via RNAi knockdown of various microtubule-organizing genes</li>
    <li>Performed centriole depletion experiments demonstrating centrosome involvement in correct angiogenic sprouting</li>
    <li>Carried out angiogenic assays, culture and live imaging of organoids, analysis of microtubule/Golgi/centrosome/mitochondrial patterning, performed genetic knockdowns, etc.</li>
  </ul>
  </section>

    <section class="cv-entry light-section-half" style="margin-bottom: 1.5rem;">
  <div class="cv-entry-header">
    <span class="cv-entry-title">Master researcher</span>
    <span class="cv-entry-years">2014-2015</span>
  </div>
  <div class="cv-entry-subtitle-1">
    Molecular technique development (molecular epigenetics)
  </div>
  <div class="cv-entry-subtitle-2">
    Pediatric Gastroenterology lab<br>
    Wilhelmina Children's Hospital<br>
    UMC Utrecht
  </div>
   <ul class="cv-entry-bullets">
    <li>Developed massively parallel <i>in vitro</i> regulatory region screen for human genomes based on STARR-seq</li>
    <li>Succeeded in identifying & quantitatively measuring DNA regulatory element (e.g. enhancer) activity across entire human genome in a single experiment</li>
    <li>Amplified entire human genome with 20× library depth and full complexity <i>in vitro</i></li>
    <li>Routine assistance with cell passaging, cDNA library preparations, qRT-PCR measurements, etc.</li>
  </ul>
  </section>

    <section class="cv-entry">
  <div class="cv-entry-header">
    <span class="cv-entry-title">Bachelor research internship</span>
    <span class="cv-entry-years">2013</span>
  </div>
  <div class="cv-entry-subtitle-1">
    Molecular cell biology research into insect antennae and olfaction
  </div>
  <div class="cv-entry-subtitle-2">
    Fungal Physiology / de Vries lab<br>
    Westerdijk Fungal Biodiversity Institute
  </div>
   <ul class="cv-entry-bullets">
   <li>Profiled bioactivities (growth, secretion, enzyme activity) of various thermophilic fungal species (incl. sexual crosses) on wide variety of complex plant substrates, cultured in both solid and liquid phases</li>
   <li>Carried out spectrophotometric CAZyme assays, silver staining of fungal proteomes, and cultured various fungi on varieties of substrates</li>
  </ul>
  </section>
</section>

<section class="mainfont" id="education">
<h1>EDUCATION</h1>
<div class="hz-divider" style="margin-top: 0rem; margin-bottom: 1.5rem;"></div>
  <div class="degree-entry">
    <div class="degree-left"><mark>PhD</mark></div>

    <div class="degree-right">
      <div><b>Molecular Biology & Neuroscience</b></div>
      <div><i>Magna cum laude</i></div>
      <div><a href="https://www.uni-jena.de/en" target="_blank">Friedrich Schiller University Jena</a> <img src="https://flagcdn.com/w20/de.png" alt="DE"></div>
      <div><a href="https://www.ice.mpg.de/129183/imprs" target="_blank">International Max Planck Research School</a> (Max Planck Society)</div>
    </div>
  </div>

  <div class="degree-entry">
    <div class="degree-left"><mark style="border: 1px solid gainsboro;">MSc</mark></div>

    <div class="degree-right">
      <div><b>Molecular and Cellular Life Sciences</b></div>
      <div><i>Research Master degree<br>
      3.93 GPA</i></div>
      <div><a href="https://www.uu.nl/en" target="_blank">Utrecht University</a> <img src="https://flagcdn.com/w20/nl.png" alt="NL" style="position: relative; top: 1px;"></div>
    </div>
  </div>

  <div class="degree-entry">
    <div class="degree-left"><mark style="border: 1px solid brown;">BSc</mark></div>

    <div class="degree-right">
      <div><b>Liberal Arts & Sciences</b></div>
      <div><i>Major: life sciences, neuroscience, biotechnology<br>
      Minor: psychology</i><br>
      <i>3.7 GPA</i></div>
      <div><a href="https://www.uu.nl/en/organisation/university-college-utrecht" target="_blank">University College Utrecht</a> <img src="https://flagcdn.com/w20/nl.png" alt="NL" style="position: relative; top: 1px;"></div>
    </div>
  </div>

</section>

<!-- consider adding <br> or spacers to just give the place some space -->

<section class="mainfont" id="academic-works">
<h1>ACADEMIC WORKS:</h1>
<div class="hz-divider" style="margin-top: 0rem; margin-bottom: 1.5rem;"></div>
  <h3 style="text-align:center;">@ Doctorate</h3>
  <section style="all: revert; padding-bottom: 0.5rem;" class="vlined-section">
  <!-- <div class="vline-layout"> -->
  <!-- <div class="vline"></div> -->
  <!-- <div class="vline-content"> -->
        <section class="light-section" style="font-size: 1rem">
        <details class="collapse">
        <summary><b>Doctoral dissertation</b><br>Non-canonical sensory biology of antennal cells in insect olfaction</summary>
        <p class="blockexplanation-black" style="font-size: 0.9rem">
        <b style="color: white;">Title:</b><br>
        Non-canonical sensory biology of antennal cells in insect olfaction<br>
        <b style="color: white;">Type:</b> Doctoral dissertation<br>
        <b style="color: white;">Year:</b> 2024<br>
        <b style="color: white;">Doctoral advisors:</b><br>
        Prof. Bill S. Hansson<br>
        Prof. Stefan Heinemann<br>
        <b style="color: white;">Submitted to:</b><br>
        Faculty of Biological Sciences<br>
        Friedrich Schiller University Jena<br>
        </p>
        <p class="blockexplanation" style="font-size: 0.9rem">
        This work is a treatise on the underexplored aspects of insect olfaction. It focuses on non-neuronal cells and mechanisms beyond that of olfactory receptor-mediated sensory reception.<br>
        Included: an extensive review, several original research manuscripts, and a detailed discussion on the present and future state of the field.
        </p>
        Published <a href="https://doi.org/10.22032/dbt.62655" target="_blank">here</a>.
        </details>
        </section>
  <!-- </div> -->
  <!-- </div> -->
  </section>

  For a list of articles published in scientific journals, see <a href="publications.html">here</a>.


<div class="hz-divider"></div>
<h3 style="text-align:center;">@ Master</h3>

  <section style="all: revert;" class="vlined-section">
  <!-- <div class="vline-layout">
  <div class="vline"></div>
  <div class="vline-content"> -->
<section class="light-section" style="font-size: 1rem">
        <details class="collapse">
        <summary><b>Master thesis</b><br>Physiological functions of abundant extracellular chaperones</summary>
        <p class="blockexplanation-black" style="font-size: 0.9rem">
        <b style="color: white;">Full title:</b><br>
        Physiological functions of abundant extracellular chaperones: involvement of secreted chaperones in proteostasis, 
        immunity and protein deposition diseases<br>
        <b style="color: white;">Type:</b> Master thesis<br>
        <b style="color: white;">Year:</b> 2016<br>        
        <b style="color: white;">Supervision:</b><br>
        Dr. Martina Radli<br>
        Prof. Stefan Rudiger<br>
        <b style="color: white;">Location:</b><br>
        Dept. Cellular Protein Chemistry<br>
        Faculty of Science<br>
        Utrecht University
        </p>
        <p class="blockexplanation" style="font-size: 0.9rem">In this work I review disconnected literature concerning molecular
        chaperones (proteins that sequester and help other proteins fold correctly). In particular, I focus on the set of chaperones known
        to function <i>outside the cell</i>.<br>Though such chaperones are extremely abundant in bodily fluids and implicated in health and disease, they remain poorly characterized,
        and this work aims to be a first unifying review comparing (dis)similarities among them. I argue that
        extracellular chaperones present as attractive, imperative avenues for further biomedical research, and as targets for future medical interventions.</p>
        <p style="font-size: 0.9rem">You can request the original work <a href="https://www.researchgate.net/publication/355154404_Physiological_functions_of_abundant_extracellular_chaperones" target="_blank">here</a>.
        </p>
        </details>
        </section>

<h5 style="text-align:center;">Research projects</h5>
        <section class="light-section" style="font-size: 1rem">
        <details class="collapse">
        <summary><b>Project</b><br>Microtubule/MTOC dynamics in sprouting angiogenesis</summary>
        <p class="blockexplanation-black" style="font-size: 0.9rem">
        <b style="color: white;">Full title:</b><br>
        Microtubule involvement in sprouting angiogenesis: Role of microtubule minus end-binding protein CAMSAP2 and MTOC organelles in endothelial spheroid model system<br>
        <b style="color: white;">Type:</b> Master Research Project<br>
        <b style="color: white;">Year:</b> 2015-2016<br>
        <b style="color: white;">Supervision:</b><br>
        Dr. Maud J. Martin<br>
        Prof. Anna Akhmanova<br>
        <b style="color: white;">Location:</b><br>
        Cellular Dynamics (Akhmanova) lab<br>
        Dept. Cell Biology<br>
        Faculty of Science<br>
        Utrecht University
        </p>
        <p class="blockexplanation" style="font-size: 0.9rem">
        In this year-long project, I used endothelial spheroids made from human umbilical vein endothelial cells (HUVECs)
        to model and induce blood vessel outgrowth (i.e. sprouting angiogenesis). In particular, I focused on 
        studying which role microtubule, microtubule end-binding proteins and microtubule-organizing centers (MTOCs) 
        such as the Golgi apparatus and centrosomes have in the sprouting process. Furthermore, with RNAi, I knocked 
        out CAMSAP2, a microtubule minus end-stabilizing protein, and showed abberant sprouting and Golgi organization. 
        I also performed a lot of cytological quantitation using widefield and matrix-embedded live imaging, 
        including counting mitochondria, looking for patterning in elements of the cytoskeleton and organelles.
        </p>
        </details>
        </section>

        <section class="light-section" style="font-size: 1rem">
        <details class="collapse">
        <summary><b>Project</b><br>
        Developing STARR-seq for the human genome</summary>
        <p class="blockexplanation-black" style="font-size: 0.9rem">
        <b style="color: white;">Full title:</b><br>
        Application of STARR-Seq to the human genome: a genome-wide DNA regulatory region screen<br>
        <b style="color: white;">Type:</b> Master research Project<br>
        <b style="color: white;">Year:</b> 2014-2015<br>
        <b style="color: white;">Supervision:</b><br>
        Dr. Michal Mokry<br>
        <b style="color: white;">Location:</b><br>
        Pediatric Gastroenterology<br>
        Laboratory of Translational Immunology<br>
        Wilhelmina Children's Hospital<br>
        UMC Utrecht
        </p>
        <p class="blockexplanation" style="font-size: 0.9rem">
        For over a year, in this project I adapted a new high-throughput technique (STARR-seq) 
        through which we can identify and measure the activity of DNA regulatory elements (e.g. enhancers). 
        Specifically, the method was limited to work only on smaller genomes and required exhaustive cloning efforts 
        to obtain enough DNA of interest at sufficient library complexity and depth for assaying. 
        I discovered a way to circumvent this requirement by preparing suffient "input" DNA <i>in vitro</i>. 
        I show the transcriptional activity readout is of sufficient complexity, covering the entire 
        human genome in terms of readout information; additionally, I compare the result with 
        golden standard assay results (luciferase reporter assays), showing that the novel adapted methodology 
        is cheaper, quicker, and now viable for human (and similarly large) genomes. 
        Reading transcriptional activity <i>en masse</i> allows to identify and quantitate DNA regulatory element activity 
        in a single experimental run, which in turn helps discover the critical roles DNA elements play in 
        cell type specification and disease pathogenesis.
        </p>
        </details>
        </section>

<h5 style="text-align:center;">Lab rotations</h5>
        <section class="light-section" style="font-size: 0.9rem">
        <details class="collapse">
        <summary>Computational Structural Biology</summary>
        <p class="blockexplanation-black" style="font-size: 0.9rem">
        <b style="color: white;">Year:</b> 2013<br>
        <b style="color: white;">Supervision:</b><br>
        Dr. Joao Rodriguez<br>
        Prof. Alexandre Bonvin<br>
        <b style="color: white;">Location:</b><br>
        Bonvin lab<br>
        Computational Structural Biology group<br>
        Utrecht University</p>
        <p class="blockexplanation" style="font-size: 0.9rem">
        Training: various docking simulations, modeling of biomolecular complexes, HADDOCK software, PyMOL, Bash scripting, Scientific Linux
        </p>
        </details>
        </section>

        <section class="light-section" style="font-size: 0.9rem">
        <details class="collapse">
        <summary>Molecular Oncology</summary>
        <p class="blockexplanation-black" style="font-size: 0.9rem">
        <b style="color: white;">Year:</b> 2014<br>
        <b style="color: white;">Supervision:</b><br>
        Prof. Paul van Bergen en Henegouwen<br>
        <b style="color: white;">Location:</b><br>
        Molecular Oncology lab<br>
        Dept. Cell Biology<br>
        Faculty of Science<br>
        Utrecht University
        </p>
        <p class="blockexplanation" style="font-size: 0.9rem">
        Training: EGFR signaling/internalization into endosomes/multivesicular bodies, use of nanobodies <i>in vitro</i>
        </p>
        </details>
        </section>

  <!-- </div>
  </div> -->
  </section>

<div class="hz-divider"></div>
  <h3 style="text-align:center;">@ Bachelor</h3>   

  <section style="all: revert;" class="vlined-section">
  <!-- <div class="vline-layout">
  <div class="vline"></div>
  <div class="vline-content"> -->

        <section class="light-section" style="font-size: 1rem">
        <details class="collapse">
        <summary><b>Bachelor thesis</b><br>Fungal physiology of thermophiles and selected progeny grown on complex substrates</summary>

        <p class="blockexplanation-black" style="font-size: 0.9rem">
        <b style="color: white;">Title:</b><br>
        Saccharification capacity of the industrially relevant thermophile <i>Myceliophthora heterothallica</i> by comparing parental strains with progeny selected ofr growth on spruce biomass<br>
        <b style="color: white;">Supervision:</b><br>
        Dr. Joost van den Brink<br>
        <b style="color: white;">Location:</b><br>
        Westerdijk Institute<br>
        <i>(formerly: CBS-KNAW Fungal Biodiversity Centre)</i>
        </p>
        
        <p class="blockexplanation" style="font-size: 0.9rem">
        The thesis subject includes third-gen (lignocellulosic) biofuel production and the potential use of thermophilic 
        fungi in degrading complex plant matter as a means of improving biofuel production. In this context, I 
        present experimental evidence and a research report involving testing of the sugar-releasing capacity in strains 
        and growth-selected progeny of industrially-relevant thermophiles, grown on 7 complex plant biomasses. Experiments involve growth 
        profiling of thermophilic, filamentous fungi cultured in both solid and liquid phase, sexually crossing them, selecting catabolic strains,
        and testing their CAZyme activity and degree of enzyme production in various ways. Essentially, the aim of this project was to help further 
        reveal whether biofuel generation from non-food crops is industrially viable.
        </p>
        </details>
        </section>

        <section class="light-section" style="font-size: 0.9rem">
        <details class="collapse">
        <summary>Laboratory training courses</summary>
        <p style="font-size: 0.9rem">
        - Cellular signaling<br>
        - Biochemical pharmacology<br>
        - Visual psychophysics
        </p>
        </details>
        </section>

  <!-- </div>
  </div> -->
  </section>
    </section>
<!-- <div class="hz-divider"></div> -->

<section class="mainfont" id="skills">
  <h1>SKILLS</h1>
  <div class="hz-divider" style="margin-top: 0rem; margin-bottom: 1.5rem;"></div>

<h3>SOFT SKILLS</h3>
<section class="skill-entry">
  <div class="skill-left">General</div>
  <div class="skill-right">
    Research<br>
    Project management<br>
    Scientific/technical writing<br>
    Public speaking, oral presentation<br>
    Teaching and guided supervision
  </div>
</section>
<section class="skill-entry">
  <div class="skill-left">Technical</div>
  <div class="skill-right">
    Documentation, data management<br>
    Multilingual translation<br>
    Community organization <span style="font-size: 0.85rem; font-style: italic;">(conferences, representation, recruitment, etc.)</span>
  </div>
</section>

<h3>LABORATORY</h3>
<section class="skill-entry">
  <div class="skill-left">Wet Lab</div>
  <div class="skill-right">
    Cell culture <i>(bacterial, fungal, animal, human)</i><br>
    2D (plate) culture<br>
    3D (organoid) culture<br>
    Molecular cloning<br>
    Organelle isolation (e.g. nuclei)<br>
    DNA/RNA/protein isolation<br>
    Transfection (pro/eukaryotic), e.g. via electroporation, liposome, etc.<br>
    Immunohisto/cytochemistry<br>
    Fine tissue dissection<br>
    Cryotomy
  </div>
</section>
<section class="skill-entry">
  <div class="skill-left">Molecular bio</div>
  <div class="skill-right">
    Nucleic acid & protein chemistry:<br>
    (gel electrophoresis, blotting, silver staining, plasmid work, etc.)<br>
    PCR<br>
    qRT-PCR<br>
    FISH<br>
    DNA/RNA library prep<br>
    Plasmid prep    
  </div>
</section>
<section class="skill-entry">
  <div class="skill-left">Microscopy</div>
  <div class="skill-right">
  Various slide prep and histochemistry experience<br>
  Widefield and confocal (fluorescence) microscopy (SD and LS)<br>
  Long-term live specimen imaging<br>
  Super-resolution imaging<br>
  Ca<span class="sup">2+</span> and K<span class="sup">+</span> imaging<br>
  Photoactivation, photobleaching<br>
  etc.
  </div>
</section>
<section class="skill-entry">
  <div class="skill-left">Electrophysiology</div>
  <div class="skill-right">
    Whole-cell patch clamp<br>
    Single-sensillum recording (SSR)<br>
    Electroantennography
  </div>
</section>
<section class="skill-entry">
  <div class="skill-left">Animal work</div>
  <div class="skill-right">
    <i>Drosophila</i> genetics, crossing and breeding<br>
    Rudimentary mouse work
  </div>
</section>

<h3>STATISTICS & DATA ANALYSIS</h3>
<section class="skill-entry">
  <div class="skill-left">Statistics</div>
  <div class="skill-right">
    Biostatistics<br>
    Quantitative and qualitative analysis<br>
    Statistical inference
  </div>
</section>

<section class="skill-entry">
  <div class="skill-left">Data work</div>
  <div class="skill-right">
    Transcriptomic analysis: bulk and single-cell/nucleus datasets<br>
    Feature extraction<br>
    Image processing<br>
    Automation / data pipelining<br>
    Context-dependent analytics<br>
    Advanced data graphics, plotting, visualization
  </div>
</section>

<h3>CODING</h3>
<section class="skill-entry">
  <div class="skill-left">Statistical programming</div>
  <div class="skill-right">
    R<br>
    Python<br>
  </div>
</section>
<section class="skill-entry">
  <div class="skill-left">Other</div>
  <div class="skill-right">
    git<br>
    bash<br>
    LaTeX<br>
    SQL<br>
    HTML<br>
    CSS<br>
    JavaScript<br>
  </div>
</section>

<h3>TECHNICAL PC SKILLS</h3>
<section class="skill-entry">
  <div class="skill-left">General purpose</div>
  <div class="skill-right">
    MS Office suite<br>
    Overleaf<br>
    Adobe Illustrator<br>
    Adobe Photoshop<br>
    Reference managers <span style="font-size: 0.85rem; font-style: italic;">(Mendeley, Zotero, EndNote)</span>
  </div>
</section>
<section class="skill-entry">
  <div class="skill-left">Special purpose</div>
  <div class="skill-right">
      FIJI/ImageJ<br>
      GraphPad<br>
      ChemDraw<br>
      TILLVision<br>
      BioRender<br>
      MATLAB<br>
      SPSS<br>
      PyMOL<br>
      Wolfram Mathematica<br>
      Various genome browsers<br>
      ZEN Imaging suite<br>
      VS Code<br>
      Autospike<br>
      <i>etc.</i>
  </div>
</section>

<h3>MEDIA PRODUCTION</h3>

<section class="skill-entry">
  <div class="skill-left">Video</div>
  <div class="skill-right">
    Davinci Resolve<br>
    (Sony) Vegas Pro<br>
    Open Broadcasting Software (OBS)
  </div>
</section>
<section class="skill-entry">
  <div class="skill-left">Audio</div>
  <div class="skill-right">
    Audacity<br>
    Various DAW experiences <span style="font-size: 0.85rem; font-style: italic;">
      (Ableton Live, FL Studio, Nuendo, Pro Tools, etc.)</span>
  </div>
</section>


<h3>LANGUAGES</h3>
<section class="skill-entry">
  <div class="skill-left">Native level</div>
  <div class="skill-right">
    English<br>
    Polish<br>
    Czech<br>
    Serbo-Croatian
  </div>
</section>
<section class="skill-entry">
  <div class="skill-left">Intermediate level</div>
  <div class="skill-right">
    German<br>
    Dutch
  </div>
</section>

</section>

<section  id="awards">
  <h1>AWARDS</h1>
<div class="hz-divider" style="margin-top: 0rem; margin-bottom: 1.5rem;"></div>
  <ul class="mainfont fa-awards" style="font-size: 1rem"><li>IMPRS recruit, grantee & associate 2018-2024 (<a href="https://www.ice.mpg.de/312631/graduates?year=2024" target="_blank">link</a>)</li> 
      <li>ECRO stipend awardee 2021 (<a href="https://www.ecro.online/grants/awardees/" target="_blank">link</a>)</li>
      <li>Full scholarship 2010-2013, University College Utrecht</li>
      <li>Duke of Edinburgh's Award (Gold)</li></ul>
</section>

<section class="mainfont" id="teaching">
<h1>TEACHING</h1>
<div class="hz-divider" style="margin-top: 0rem; margin-bottom: 1.5rem;"></div>
  <ul>
    <li><b>Student supervision:</b> 3 undergraduate and master students on respective experimental projects (2018-2020)</li>
    <li><b>Co-teaching undergraduate practical lab course 'Olfactory Neurogenetics'</b><br>
      (25 students, 2018, Friedrich Schiller University Jena)</li>
  </ul>
</section>

<section class="mainfont" id="volunteering">
<h1>VOLUNTEER WORK</h1>
<div class="hz-divider" style="margin-top: 0rem; margin-bottom: 1.5rem;"></div>
    <h3>conference organization</h3>
      As Chief Organizer:
      <ul>
        <li>Max Planck Society's <b>PhDnet General Meeting 2021</b> <i>(200+ attendees)</i></li>
        <li><b>2019 Scientific retreat</b> for MPI department & guests <i>(50+ attendees)</i></li>
      </ul>
      
    <h3>science outreach</h3>    
      <ul>
        <li><b>Former member, development advisor</b> for open science educational platform <a href="https://enabla.com/en/" target="_blank" rel="noopener noreferrer">Enabla</a></li>
        <li><b>Biology Stack Exchange contributor</b> (top 1% user)</li>
        <li><b>Podcasting:</b> <a href="https://www.phdnet.mpg.de/offspring/podcast" target="_blank" rel="noopener noreferrer">Offspring Magazine Podcast</a> and local early career researcher podcast</li>
        <li><b>Podcast support:</b> Setup & assistance for PR dept. @ Max Planck Institute for Chemical Ecology</li>
      </ul>

    <h3>representation</h3>
      <ul>
        <li><b>3× elected representative of doctoral researchers</b><br>(Max Planck Institute for Chemical Ecology & IMPRS programme)</li>
        <li>2× invited <b>supervisor for PhD student recruitment</b><br>(invited by MPICE IMPRS department)</li>
      </ul>
    
    <h3>as lifelong translator</h3>    
      Multilingual translator <i>(Polish, Czech, Serbo-Croatian, English)</i> with various experiences
      <ul>
        <li>Prague Polish/Swedish embassy, consulate and cultural institute correspondences</li>
        <li><a href="https://www.chess.com/" target="_blank" rel="noopener noreferrer">Chess.com</a></li>
        <li>concert and festival material</li>
        <li>techincal documents, product data sheets</li>
        <li>private business correspondences</li>
        <li>etc.</li>
      </ul>

    <h3>private teaching, open lectures</h3>    
      <ul>
        <li>Chess teacher (2016-2017) (children, adolescents, both OTB + online play)</li>
        <li>Public science lecture series speaker (2016-2017)</li>
      </ul>

    <h3>tourguiding</h3>    
      <i>Airbnb experiences</i> - Prague district tour guide (2016-2017) 

    <h3>arts and sports</h3>     
      <ul>
        <li>Bassist</li>
        <li>Bandleader
          <ul>
            <li>ICEbreakers</li>
            <li>JAFFI</li>
          </ul>
        </li>
        <li>Filmmaker</li>
        <li>Sports coaching</li>
        <li>Competition analytics for online communities</li>
      </ul>

</section>

</div>
</body>

<footer class="site-footer">
  <div class="footer-layout">
    <!-- LEFT SIDE ICONS -->
    <span style="color: #ffbf00; opacity: 0.75;">CONTACT ME:</span>
    <div class="footer-left social">
      <a href="mailto:sinprelic@yahoo.com"><i class="fa-solid fa-envelope"></i></a>
      <a href=""><i class="fa-solid fa-phone"></i></a>
    </div>
    <!-- CENTER TEXT -->
    <div class="footer-center">
      handwritten in vanilla html, css and js<br>
      © 2026 S. PRELIC — ALL RIGHTS RESERVED.
    </div>
    <!-- RIGHT SIDE ICONS -->
    <span style="color: #ffbf00; opacity: 0.75;">MY SOCIALS:</span>
    <div class="footer-right social">
      <a href="https://www.linkedin.com/in/sinprelic/" target="_blank" rel="noopener noreferrer"><i class="fa-brands fa-linkedin"></i></a>
      <a href="https://orcid.org/0000-0002-6494-7389" target="_blank" rel="noopener noreferrer"><i class="fa-brands fa-orcid"></i></a>
      <a href="https://www.researchgate.net/profile/Sinisa-Prelic" target="_blank" rel="noopener noreferrer"><i class="fa-brands fa-researchgate"></i></a>
      <a href="https://scholar.google.com/citations?user=8UJ2xZQAAAAJ" target="_blank" rel="noopener noreferrer"><i class="fa-brands fa-google-scholar"></i></a>
      <a href="https://loop.frontiersin.org/people/1360472/overview" target="_blank" rel="noopener noreferrer"><i class="fa-solid fa-microscope"></i></a>
      <a href="https://github.com/sinprelic" target="_blank" rel="noopener noreferrer"><i class="fa-brands fa-github"></i></a>
      <a href="https://linktr.ee/sinprelic" target="_blank" rel="noopener noreferrer"><i class="fa-brands fa-linktree"></i></a>
    </div>
  </div>
</footer>

</html>